AgentDiscoveries-Backend/pom.xml

```diff
@@ -221,6 +260,25 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <skipTests>${skip.integration.tests}</skipTests>
+                    <systemPropertyVariables>
+                        <target.address>${target.address}</target.address>
+                    </systemPropertyVariables>
+                </configuration>
+                <version>2.20.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
```

Comment:
How do these goals cause your selenium tests to be run? 😛 There's some magic here somewhere

Reply:
The goals tells it to use the maven-failsafe-plugin for the integration-test step, in which it runs all tests of forms like /IT*.java or /*IT.java
...Now I look at it again, how does it know how to run the unit tests? 😕